### ISSUE TITLE
Update Origami Navigation Service  with updated markup

### ISSUE THREAD

**Issue by notlee** [Financial-Times/origami#907]
We've made a number of markup changes to o-header recently.
Unlike non-breaking CSS & JS, this will not roll out to users automatically.
In the future, users will be able to use TSX templates to automatically 
receive markup updates. But for now, we need users to manually update 
their markup.

These PRs reflect recent (non-breaking) markup changes:
- https://github.com/Financial-Times/origami/pull/871 
- https://github.com/Financial-Times/origami/pull/903

Their changes should be reflected in page kit, to roll out to the majority
of major ft.com projects:
https://github.com/Financial-Times/dotcom-page-kit/tree/main/packages/dotcom-ui-header

Their changes should also be reflected in the [origami navigation service](https://github.com/Financial-Times/origami-navigation-service) example.
As the service provides an example page using o-header with real, populated markup,
sometimes users copy-paste from here:
https://github.com/Financial-Times/origami-navigation-service

**Comment by notlee** [comment]
Origami Navigation Service has been updated. We'll communicate with Customer Products about the required migration to `o-header@11` and offer our support
